keps/sig-node/4727-reasonable-image-gc/README.md

```diff
@@ -0,0 +1,439 @@
+
+
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [ ] (R) Enhancement issue in release milestone, which links to KEP dir in [kubernetes/enhancements] (not the initial KEP PR)
+- [ ] (R) KEP approvers have approved the KEP status as `implementable`
+- [ ] (R) Design details are appropriately documented
+- [ ] (R) Test plan is in place, giving consideration to SIG Architecture and SIG Testing input (including test refactors)
+  - [ ] e2e Tests for all Beta API Operations (endpoints)
+  - [ ] (R) Ensure GA e2e tests meet requirements for [Conformance Tests](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md) 
+  - [ ] (R) Minimum Two Week Window for GA e2e tests to prove flake free
+- [ ] (R) Graduation criteria is in place
+  - [ ] (R) [all GA Endpoints](https://github.com/kubernetes/community/pull/1806) must be hit by [Conformance Tests](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md) 
+- [ ] (R) Production readiness review completed
+- [ ] (R) Production readiness review approved
+- [ ] "Implementation History" section is up-to-date for milestone
+- [ ] User-facing documentation has been created in [kubernetes/website], for publication to [kubernetes.io]
+- [ ] Supporting documentation—e.g., additional design documents, links to mailing list discussions/SIG meetings, relevant PRs/issues, release notes
+
+<!--
+**Note:** This checklist is iterative and should be reviewed and updated every time this enhancement is being considered for a milestone.
+-->
+
+[kubernetes.io]: https://kubernetes.io/
+[kubernetes/enhancements]: https://git.k8s.io/enhancements
+[kubernetes/kubernetes]: https://git.k8s.io/kubernetes
+[kubernetes/website]: https://git.k8s.io/website
+
+## Summary
+
+the default value of `--image-gc-high-threshold` option is 85%, and the default value of `imagefs.available` option is 
+less than 15%, this will result in image garbage collection not taking effect until node gets disk pressure 
+
+
+## Motivation
+
+There is no standard to judge whether the value of `--image-gc-high-threshold` is reasonable in different scenarios
+
+### Goals
+
+discuss reasonable values of `--image-gc-high-threshold` for different scenarios, and constrain them by some means.
+
+
+### Non-Goals
+
+
+## Proposal
+
+
+
+### User Stories (Optional)
+
+
+#### Story 1
+
+In big data computing scenarios, we often run some computing tasks. 
+When these tasks are completed, a large number of images are stored on the node. 
+At this time, we want to perform image garbage collection before the node disk pressure occurs.
+
+#### Story 2
+
+Keep the current usage that turn off image garbage collection by setting `--image-gc-high-threshold` 100%.
+
+
+### Notes/Constraints/Caveats (Optional)
+
+
+
+### Risks and Mitigations
+
+
+## Design Details
+
+Add `ImageGCHighThresholdAccurate` feature gate to kubelet. 
```

Comment:
I don't know how informative `accurate` is here. May I suggest a feature like `ImageGCBeforeStorageEviction` or something? I think the feature should describe the function of the change